scripts/check

```diff
@@ -17,9 +17,19 @@ private_token="$(jq -r '.source.private_token // ""' < "${payload}")"
 project_path="$(jq -r '.source.project_path // ""' < "${payload}")"
 username="$(jq -r '.source.username // ""' < "${payload}")"
 password="$(jq -r '.source.password // ""' < "${payload}")"
+private_key="$(jq -r '.source.private_key // ""' < "${payload}")"
 version_sha="$(jq -r '.version.sha // ""' < "${payload}")"
 
-uri="https://${username}:${password}@${gitlab_host}/${project_path}.git"
+if [[ ! -z $private_key ]]; then
+  mkdir -p ~/.ssh/
+  id_rsa=~/.ssh/id_rsa
```

Comment:
I think this variable assignment would be clearer if the string was surrounded by single quotes.